<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bodymass</class>
 <widget class="QMainWindow" name="bodymass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Body Mass Index Calculator</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhLatinOnly</set>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="2" colspan="2">
     <widget class="QLabel" name="labelInter">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QSpinBox" name="spinBoxWeight">
      <property name="accessibleName">
       <string extracomment="kg"/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string notr="true">kg</string>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSlider" name="SliderWeight">
      <property name="maximum">
       <number>300</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Height">
      <property name="text">
       <string>Height(cm)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QSpinBox" name="spinBoxHeight">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string>cm</string>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="Gender">
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Weight">
      <property name="text">
       <string>Weight(kg)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QSlider" name="SliderHeight">
      <property name="maximum">
       <number>300</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="indexBody">
      <property name="text">
       <string>Body Mass Index</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="Interpretation">
      <property name="text">
       <string>Interpretation</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QLabel" name="labelBody">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="labelGender">
      <property name="text">
       <string>Female</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>493</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>10</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText</set>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuGender">
    <property name="title">
     <string>Gender</string>
    </property>
    <addaction name="actionFemale"/>
    <addaction name="actionMale"/>
   </widget>
   <addaction name="menuAction"/>
   <addaction name="menuGender"/>
  </widget>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionFemale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string notr="true">Female</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionMale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>SliderWeight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxWeight</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxWeight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SliderWeight</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxHeight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SliderHeight</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderHeight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxHeight</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxHeight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SliderHeight</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderHeight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxHeight</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
